releases/history.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## Version 2.0
+* ### Latest Release: [v2.0.0](https://github.com/openebs/mayastor/releases/tag/v2.0.0)
+  * released: 
+* ### Patch Releases:
```

Comment:
Don't we need to fill this space with the patch releases upto 1.0.5 ?

Reply:
This PR is for 2.0.0 Hence, as and when we''ll have patch release for 2.0 (2.0.x) I will be adding it here. Is that alright?

Reply:
I see. Ok